### DISCUSSION TITLE
Peak size

### DISCUSSION THREAD

**Discussion by koh2ng0** [GreenleafLab/ArchR#1878] [Questions / Documentation]
Hi, all

I analyzed the single-cell ATAC-seq dataset using ArchR, and I have a question about peak size. As far as I know, the average size of the peak is ~500bp usually, but in my case, peaks that I extracted with ArchR are up to ~20kbp. I looked it up and found out that if it uses MACS2 for peak calling, it may lead to expanding peak size. So can I have this problem when I use ArchR? And I wonder if it's normal to appear in tens of kbp like this.

Thanks.

**Comment by rcorces** [comment]
This is not possible. Peak sets generated through ArchR strictly have 501 bp peaks.
